common/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/common/model/PrisonerTest.kt

```diff
@@ -0,0 +1,77 @@
+package uk.gov.justice.digital.hmpps.prisonersearch.common.model
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+
+class PrisonerTest {
+  @Test
+  fun `all Prisoner fields are nullable`() {
+    assertDoesNotThrow {
+      jacksonObjectMapper().readValue("{}", Prisoner::class.java)
+    }
+  }
```

Comment:
These tests check that all Prisoner attributes are nullable, including all nested objects